src/components/richText/imageNode/ImageNode.tsx

```diff
@@ -0,0 +1,200 @@
+import {
+  $applyNodeReplacement,
+  DOMConversionMap,
+  DOMConversionOutput,
+  DOMExportOutput,
+  EditorConfig,
+  DecoratorNode,
+  LexicalNode,
+  NodeKey,
+  SerializedLexicalNode,
+  Spread,
+} from "lexical";
+
+import * as React from "react";
```

Comment:
I don't believe you need these React imports in ImageNode.tsx and LexicalImage.tsx, since you omitted it from LexicalRichText.tsx